_maps/Pillar_of_Spring.json

```
@@ -0,0 +1,8 @@
+{
+    "map_name": "Pillar of Spring",
+    "map_path": "map_files/Pillar_of_SPring",
+    "map_file": "TGS_Pillar_of_Spring.dmm",
+	"traits": [{"Marine Main Ship": true}], 
+    "shuttles": {
+    }
+}
```

```suggestion
}

```
Another little line at the end of the file. Not as relevant.